server/lorax_server/utils/sgmv.py

```diff
@@ -10,6 +10,7 @@
     import punica_kernels as _kernels
 
     HAS_SGMV = not bool(os.environ.get("DISABLE_SGMV", ""))
+    print("HAS_SGMV: ", HAS_SGMV)
```

Comment:
Please remove debug code.

Reply:
still a draft! was building to deploy and make sure it was propagated, i can't get it to build on my laptop so...